server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/issue/IssueLifecycle.java

```diff
@@ -64,9 +64,32 @@ public void initNewOpenIssue(DefaultIssue issue) {
     issue.setStatus(Issue.STATUS_OPEN);
     issue.setEffort(debtCalculator.calculate(issue));
   }
-  
-  public void updateExistingOpenissue(DefaultIssue base) {
-    // nothing to do
+
+  /**
+   * Instead of merging the issue ({@link #mergeExistingOpenIssue(DefaultIssue, DefaultIssue)}), it creates a new entry in the 
+   * database so that the issue has it's own new lifecycle.
+   */
+  public void copyExistingOpenIssue(DefaultIssue raw, DefaultIssue base) {
+    raw.setNew(false);
+    raw.setKey(Uuids.create());
+    raw.setType(base.type());
+    raw.setCreationDate(base.creationDate());
+    raw.setUpdateDate(base.updateDate());
+    raw.setCloseDate(base.closeDate());
+    raw.setResolution(base.resolution());
+    raw.setStatus(base.status());
+    raw.setAssignee(base.assignee());
+    raw.setAuthorLogin(base.authorLogin());
+    raw.setTags(base.tags());
+    raw.setAttributes(base.attributes());
+    raw.setEffort(debtCalculator.calculate(raw));
+    raw.setOnDisabledRule(base.isOnDisabledRule());
+    if (base.manualSeverity()) {
+      raw.setManualSeverity(true);
+      raw.setSeverity(base.severity());
+    }
+    // TODO do we need to copy something related to component or module from base?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)